src/state.ts

```diff
@@ -49,6 +50,7 @@ browser.storage.local
 const state = (new Proxy(overlay, {
     /** Give defaults if overlay doesn't have the key */
     get(target, property) {
+        // if (notBackground()) throw "State object must be accessed with getAsync in content"
```

Comment:
IMO we shouldn't have commented code. If this is for debugging purpose, this should use the logger instead.

Reply:
Ah, good catch. This is not supposed to be commented out. The trouble is that the errors it throws can't be traced back to where they came from, so finding everywhere that tries to access it is a pain.

Reply:
`throw new Error("...")` should give you a trace, unless webpack mangles that?

Reply:
Yeah, it should... and it does. I have no idea what I was doing yesterday.